examples/https/package.json

```diff
@@ -32,15 +32,15 @@
     "url": "https://github.com/open-telemetry/opentelemetry-js/issues"
   },
   "dependencies": {
-    "@opentelemetry/api": "^1.0.0",
-    "@opentelemetry/exporter-jaeger": "1.9.1",
-    "@opentelemetry/exporter-zipkin": "1.9.1",
-    "@opentelemetry/instrumentation": "0.35.1",
-    "@opentelemetry/instrumentation-http": "0.35.1",
-    "@opentelemetry/resources": "1.9.1",
-    "@opentelemetry/sdk-trace-base": "1.9.1",
-    "@opentelemetry/sdk-trace-node": "1.9.1",
-    "@opentelemetry/semantic-conventions": "1.9.1"
+    "@opentelemetry/api": "1.4.1",
```

Comment:
Is this change intended? Similar in other samples.

Reply:
It was automated. No not intentional thanks

Reply:
similar as below, not sure if it is problematic and worth to work against the automation.